### PR TITLE
Test

### PR DESCRIPTION
**Pull request by johanobandoc**
update_period added, minor bugs fixed, dqn .gin files added

### CODE REVIEW

File: epsilons/Configs/rainbow_acrobot.gin [hunk 1]
```diff
@@ -15,9 +15,10 @@ JaxDQNAgent.stack_size = %gym_lib.ACROBOT_STACK_SIZE
 JaxDQNAgent.gamma = 0.99
 JaxDQNAgent.update_horizon = 3 # Rainbow
 JaxDQNAgent.min_replay_history = 500
-JaxDQNAgent.update_period = 2 
+#JaxDQNAgent.update_period = 2
```
Comment (joaogui1) [inline epsilons/Configs/rainbow_acrobot.gin]: probably should just remove it, keeping it here only brings confusion

File: epsilons/Configs/rainbow_cartpole.gin [hunk 2]
```diff
@@ -15,9 +15,10 @@ JaxDQNAgent.stack_size = %gym_lib.CARTPOLE_STACK_SIZE
 JaxDQNAgent.gamma = 0.99
 JaxDQNAgent.update_horizon = 3 # Rainbow
 JaxDQNAgent.min_replay_history = 500
-JaxDQNAgent.update_period = 2 
+#JaxDQNAgent.update_period = 2
```
Comment (joaogui1) [inline epsilons/Configs/rainbow_cartpole.gin]: Same as above

File: epsilons/agents/networks_new.py [hunk 3]
```diff
@@ -14,417 +14,453 @@
...
+    'hard_silu': jax.nn.hard_silu,
+    'hard_swish': jax.nn.hard_swish,
+    'hard_tanh': jax.nn.hard_tanh,
+    'elu': jax.nn.elu,
+    'celu': jax.nn.celu,
+    'selu': jax.nn.selu,
+    'gelu': jax.nn.gelu,
+    'glu': jax.nn.glu
+}
+'''
```
Comment (joaogui1) [inline epsilons/agents/networks_new.py]: this is just a huge commented version of what you wrote before, remove it

File: epsilons/agents/networks_new.py [hunk 4]
```diff
@@ -14,417 +14,453 @@
...
+    @nn.compact
+    def __call__(self, x):
+
+        def sample_noise(rng_input, shape):
+            noise = jax.random.normal(rng_input, shape)
+            return noise
 
-  @nn.compact
-  def __call__(self, x):
+        def f(x):
```
Comment (joaogui1) [inline epsilons/agents/networks_new.py:67]: can you add a more descriptive name?

**Review by joaogui1 — CHANGES_REQUESTED** [review]

File: epsilons/Configs/rainbow_acrobot.gin (hunk 1 above)
Comment (johanobandoc) [reply epsilons/Configs/rainbow_acrobot.gin]: Done!

File: epsilons/Configs/rainbow_cartpole.gin (hunk 2 above)
Comment (johanobandoc) [reply epsilons/Configs/rainbow_cartpole.gin]: Done!

File: epsilons/agents/networks_new.py (hunk 3 above)
Comment (johanobandoc) [reply epsilons/agents/networks_new.py]: Done!

**Pull request merged**